Cargo.toml

```diff
@@ -592,9 +592,10 @@ debug = true
 [patch.crates-io]
 # sp-io = { version = "38.0.0", git = "https://github.com/gear-tech/polkadot-sdk.git", branch = "gear-polkadot-stable2409" }
 # sp-application-crypto = { version = "38.0.0", git = "https://github.com/gear-tech/polkadot-sdk.git", branch = "gear-polkadot-stable2409" }
+
+# these patched dependecies force their `sign_ext` feature to be used by Substrate dependencies
```

Comment:
Are these patches necessary? Substrate (runtime) does not use `sign-ext`: https://github.com/paritytech/polkadot-sdk/blob/906fa9e51306635245a22e03160d1c761fae6cc3/substrate/utils/wasm-builder/src/wasm_project.rs#L850. Same goes for outdated version wasmi 0.13.2 in our Cargo.lock.